apps/pie-docs/src/_11ty/shortcodes/card.js

```diff
@@ -16,30 +20,101 @@ module.exports = function ({ items }) {
         },
     });
 
+    return isInternalLink ? internalLinkIcon : externalLinkIcon;
```

Comment:
Small optimisation, how about we only call `pieIconsSvg` once we know which one we're returning?